<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenGLWindow</class>
 <widget class="QDialog" name="OpenGLWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>962</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="MyGLWidget" name="openGLWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>752</width>
     <height>480</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_confirm">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>310</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>69</y>
     <width>141</width>
     <height>54</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_mode">
    <item>
     <widget class="QRadioButton" name="pan_radioButton">
      <property name="text">
       <string>PAN Mode</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="select_radioButton">
      <property name="text">
       <string>Select Mode</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_add_mesh">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>340</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Mesh </string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_playback">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>370</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>PlayBack</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_exit">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>430</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>150</y>
     <width>141</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_action">
    <item>
     <widget class="QRadioButton" name="radioButton_add">
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_remove">
      <property name="text">
       <string>Remove</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_adjust">
      <property name="text">
       <string>Adjust Plane</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_mesh">
      <property name="text">
       <string>Adjust Mesh</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_save">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>400</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Save All</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>myglwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
